File: 3 - Topic #3 Linked Lists/lists/Node.java
package lists;
public class Node
{
    public Integer data;
    public Node next;
    public Node prev;

    public Node(Integer data) {
        this.data = data;
        this.next = null;
        this.prev = null;
    }

    @Override
    public String toString() {
        return this.data+" ";
    }
}

File: 3 - Topic #3 Linked Lists/recipeBookProject/linked_lists/node.py
class Node:
    """The indiviual Node element for the Linked lists
    """
    def __init__(self, data: int):
        """_summary_

        Args:
            data (int): the data held in the node
        """
        self.data = data
        self.next = None
        self.prev = None
    def __repr__(self):
        """returns the string representation of the class

        Returns:
            str: the string representation of the class
        """
        return self.data + " "
    